Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -123,8 +125,13 @@ def run_checks(
           EOS
         end
 
-        skip_result = skip_conditions(formula_or_cask, json: json, full_name: full_name, quiet: quiet)
-        next skip_result if skip_result != false
+        skip_info = SkipConditions.skip_information(formula_or_cask, full_name: full_name, verbose: verbose)
+        if skip_info.is_a?(Hash)
```

Comment:
This type checking feels non-idiomatic , particularly considering it's a no-op if the type doesn't match.